demos/interactive_face_detection_demo/cpp_gapi/interactive_face_detection_gapi.hpp

```diff
@@ -56,29 +23,84 @@ static const char utilization_monitors_message[] = "Optional. List of monitors t
 
 // Set maximum FPS for playing video
 
+constexpr char help_message[] = "Print a usage message";
 DEFINE_bool(h, false, help_message);
+
+constexpr char input_video_message[] = "Required. Path to a video file (specify \"cam\" to work with camera).";
```

Comment:
Does the demo really expect cam for `-i`? If so, please change it to camera id to align with pure cpp version.

Reply:
Done